Makefile

```diff
@@ -75,7 +75,7 @@ else
     endif
 endif
 #############################################################
-ESPTOOL ?= ../tools/esptool.py
+ESPTOOL ?= ../tools/esptool.sh
```

Comment:
This looks like it may fail on Windows; maybe it belongs in the `if` statements above?